docs/cluster-guide.md

```diff
@@ -0,0 +1,90 @@
+# Guide to setting up a 3-node cluster
+
+Here is a quick how-to guide to set up a 3-node EventStoreDB cluster. It covers how to prepare the environment, generate required certificates, and how to configure and run the cluster.
+
+## Preparing the environment
+
+1. Create a folder named 'Cluster' . 
+2. In 'Cluster', create subfolders: 'Node1', 'Node2', 'Node3'.
+3. For each node, create these subfolders:
+    - 'Certificates' for storing the certificate and private key. 
+    - 'Data' for all data files.
+    - 'Index' for node indexes.
+    - 'Logs' for log files.
+4. Add a configuration file (.yml or .txt) in each node folder. 
```

Comment:
this is unclear if it's in each node folder (node 1, node 2, node 3) or each subfolder (Certificates, Data, Index, Logs). If the former, consider making this step 3 to make it clearer.

Reply:
It is the former, and good idea.